netbox_onboarding/migrations/0002_auto_20200828_1439.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.10 on 2020-08-28 14:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('netbox_onboarding', '0001_initial'),
+    ]
```

Comment:
From CI it looks like Black isn't happy with the formatting here - please run Black on this file.